networkdb/networkdb.go

```diff
@@ -64,8 +64,8 @@ type NetworkDB struct {
 	networks map[string]map[string]*network
 
 	// A map of nodes which are participating in a given
-	// network. The key is a network ID.
-	networkNodes map[string][]string
+	// network. The first key is a network ID. The second key is
```

Comment:
Is the comment "second key is" incomplete?

Reply:
yep looks like got lost